tests/models/test_dag.py

```diff
@@ -677,87 +677,176 @@ def test_following_previous_schedule(self):
+
+    def test_following_previous_schedule_daily_dag_cet_to_cest_post(self):
+        # Second pass start on
         local_tz = pendulum.timezone("Europe/Zurich")
-        start = local_tz.convert(datetime.datetime(2018, 3, 25, 2), dst_rule=pendulum.PRE_TRANSITION)
+        start_local = local_tz.convert(datetime.datetime(2018, 3, 25, 3))
+
+        # start at 3:00+01, 1:00+00
+        assert start_local.isoformat() == "2018-03-25T03:00:00+02:00"
+
+        start_utc = timezone.convert_to_utc(start_local)
+        assert start_utc.isoformat() == "2018-03-25T01:00:00+00:00"
+
+        dag = DAG("tz_dag", start_date=start_local, schedule="0 * * * *")
+
+        # run back a few
+        prev_utc = dag.previous_schedule(start_utc)
+        assert local_tz.convert(prev_utc).isoformat() == "2018-03-25T01:00:00+01:00"
+        prev_utc = dag.previous_schedule(prev_utc)
+        assert local_tz.convert(prev_utc).isoformat() == "2018-03-25T00:00:00+01:00"
+
+        # run forward a few
+        next_utc = dag.following_schedule(start_utc)
+        assert local_tz.convert(next_utc).isoformat() == "2018-03-25T04:00:00+02:00"
+        next_utc = dag.following_schedule(next_utc)
+        assert local_tz.convert(next_utc).isoformat() == "2018-03-25T05:00:00+02:00"
+
+    # def test_following_previous_schedule_daily_dag_cet_to_cest(self):
+    #     """
+    #     Make sure DST transitions are properly observed
+    #     """
+    #     local_tz = pendulum.timezone("Europe/Zurich")
+    #     start = local_tz.convert(datetime.datetime(2018, 3, 25, 2), dst_rule=pendulum.PRE_TRANSITION)
 
-        utc = timezone.convert_to_utc(start)
+    #     utc = timezone.convert_to_utc(start)
 
-        dag = DAG("tz_dag", start_date=start, schedule="0 3 * * *")
+    #     dag = DAG("tz_dag", start_date=start, schedule="0 3 * * *")
 
-        prev = dag.previous_schedule(utc)
-        prev_local = local_tz.convert(prev)
+    #     prev = dag.previous_schedule(utc)
+    #     prev_local = local_tz.convert(prev)
 
-        assert prev_local.isoformat() == "2018-03-24T03:00:00+01:00"
-        assert prev.isoformat() == "2018-03-24T02:00:00+00:00"
+    #     assert prev_local.isoformat() == "2018-03-24T03:00:00+01:00"
+    #     assert prev.isoformat() == "2018-03-24T02:00:00+00:00"
 
-        _next = dag.following_schedule(utc)
-        next_local = local_tz.convert(_next)
+    #     _next = dag.following_schedule(utc)
+    #     next_local = local_tz.convert(_next)
 
-        assert next_local.isoformat() == "2018-03-25T03:00:00+02:00"
-        assert _next.isoformat() == "2018-03-25T01:00:00+00:00"
+    #     assert next_local.isoformat() == "2018-03-25T03:00:00+02:00"
+    #     assert _next.isoformat() == "2018-03-25T01:00:00+00:00"
 
-        prev = dag.previous_schedule(_next)
-        prev_local = local_tz.convert(prev)
+    #     prev = dag.previous_schedule(_next)
+    #     prev_local = local_tz.convert(prev)
 
```

Comment:
seems something needs to be addressed here.  Should these tests be removed?  Or uncommented?  Or do we have a decision to make about what the behavior should be?

Reply:
This test actually almost passes, only fails in the final `previous_schedule` call. Since `previous_schedule` is deprecated anyway, I think it is OK to just remove that part.

I’m also converting it into a timetable test to not rely on `following_schedule`, which is also deprecated.